src/test/java/coupon/coupon/domain/DiscountTest.java

```diff
@@ -0,0 +1,96 @@
+package coupon.coupon.domain;
+
+import coupon.coupon.exception.CouponErrorMessage;
+import coupon.coupon.exception.CouponException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class DiscountTest {
+
+    @Test
+    void createDiscount_withValidPriceAndPercent_shouldSucceed() {
+        // Given
+        int validPrice = 5000;
+        int minimumOrderAmount = 50000;
+        int expectedPercent = 10;
+
+        // When
+        Discount discount = new Discount(validPrice, minimumOrderAmount);
+
+        // Then
+        assertEquals(validPrice, discount.getPrice());
+        assertEquals(expectedPercent, discount.getPercent());
+    }
+
+    @Test
+    void createDiscount_withPriceLowerThanMinimum_shouldFail() {
+        // Given
+        int invalidPrice = 500;
+        int minimumOrderAmount = 10000;
+
+        // When & Then
+        CouponException exception = assertThrows(CouponException.class, () -> {
+            new Discount(invalidPrice, minimumOrderAmount);
+        });
```

Comment:
가독성 때문에 화살표 함수 뒤에 `{}` 추가하신걸까요? `return`이 바로 나오면 생략이 가능하긴 합니다!
```suggestion
        CouponException exception = assertThrows(CouponException.class,
                () -> new Discount(invalidPrice, minimumOrderAmount));
```

Reply:
반영하였습니다 👍🙇‍♀️